### PR TITLE
[6.0] Remove excessive spaces from XML doc comments

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

**Comment by ajcvickers** [comment]
@dotnet/efteam You asked for it! But now you have 1000 files to review. 🚎

**Comment by ajcvickers** [comment]
@Pilchie FYI; changes XML docs only.

File: src/EFCore.Relational/Diagnostics/DbCommandInterceptor.cs [hunk 1]
```diff
@@ -22,20 +22,20 @@ public abstract class DbCommandInterceptor : IDbCommandInterceptor
...
         /// </summary>
-        /// <param name="eventData"> Contextual information about the command and execution. </param>
+        /// <param name="eventData">Contextual information about the command and execution.</param>
         /// <param name="result">
         ///     Represents the current result if one exists.
         ///     This value will have <see cref="InterceptionResult{DbCommand}.HasResult" /> set to <see langword="true" /> if some previous
         ///     interceptor suppressed execution by calling <see cref="InterceptionResult{DbCommand}.SuppressWithResult" />.
         ///     This value is typically used as the return value for the implementation of this method.
-        /// </param>
+        ///</param>
```
Comment (roji) [inline src/EFCore.Relational/Diagnostics/DbCommandInterceptor.cs:31]: Preposterous!

Seems like your regex was a bit over-zealous, there are various others like this (also with `</returns>` below)

**Review by roji — APPROVED** [review]
Looks great but see the comment below

**Comment by roji** [comment]
BTW I think we also discussed un-indenting, so:

```xml
<summary>
    Foo
</summary>
```

would become:

```xml
<summary>
Foo
</summary>
```

File: src/EFCore.Relational/Diagnostics/DbCommandInterceptor.cs (hunk 1 above)
Comment (ajcvickers) [reply src/EFCore.Relational/Diagnostics/DbCommandInterceptor.cs]: Updated with these fixed.

**Comment by roji** [comment]
@ajcvickers judging from the emoji positive/negative balance on the last comment, we should absolutely do this change!

On a more serious note though, this is how I'm generally used to it in .NET (e.g. https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq/src/System/Linq/IIListProvider.cs)

**Comment by ajcvickers** [comment]
🐤 🌵 🍰 🗡️ 😮‍💨 

I am the walrus.

**Comment by Pilchie** [comment]
Ok

**Pull request merged**